Initium-Odp/war/odp/ajax_trade.jsp

```diff
@@ -117,7 +117,7 @@
         <div class="trade-boxes" style="margin-top: 20px;">
         <div class='main-splitScreen'>
         <div class='${characterTradeWindowClass}'><h4>Your offer</h4>
-        <h4><a onclick='tradeSetGoldNew(event,<%=+tradeObject.getDogecoinsFor(character.getKey())%>)'>Gold: <span id='myTradeGoldAmount'><%=GameUtils.formatNumber(tradeObject.getDogecoinsFor(character.getKey())) %></span></a></h4>
+        <h4><a onclick='tradeSetGoldNew(event,<%=+tradeObject.getDogecoinsFor(character.getKey())%>, <%=character.getProperty("dogecoins")%>)'>Gold: <span id='myTradeGoldAmount'><%=GameUtils.formatNumber(tradeObject.getDogecoinsFor(character.getKey())) %></span></a></h4>

```

Comment:
Since you're not using the number value for if tests, you can make this number a bit prettier and easier to read with this: GameUtils.formatNumber(number)